docs/getting_started/components/sip.md

```diff
@@ -0,0 +1,32 @@
+# SIP
+
+Makes a SIP connection with specified provider.
+From this connection it consumes a remote RTP stream with G711 A-law audio
+and publishes through this connection all audio tracks from engine.
+
+Supports OPUS streams only.
```

Comment:
I'd get rid of this sentence. It may confuse readers since we're talking about G711 right above. Moreover, us using OPUS is, right now, strictly an engine thing which users do not necessarily need to know -- it isn't relevant here as e.g. in the File endpoint, where the file *needs* to be opus-encoded